commands/images/reddit.js

```diff
@@ -0,0 +1,75 @@
+const { MessageEmbed, MessageActionRow, MessageButton, Guild, GuildChannel, Channel, Message } = require('discord.js');
+const got = require("got")
+module.exports = {
+	name: 'reddit',
+	description: 'Shows images from a subreddit.',
+	usage: 'reddit <search query>',
+	cooldown: '10',
+	args: true,
+	aliases: ['red', 'reddit'],
+	async execute(client, message, args, functions) {
+		let subName = args.at(0);
+		console.log(args, args.length);
+		if (args.length > 1) {
+			return message.reply(functions.simpleEmbed('Please provide a single subreddit!'));
+		}
+		let subredditUrl = await got(`https://www.reddit.com/r/${subName}.json`).json()
+		let subreddit = subredditUrl.data.children
+// stealing mara's code, hopefully works
+// making x = 1 since most subreddits have a shitty pinned post. could i make it not show the pinned post? probably. do i care enough to do that? no
+let x = 1;
+		let imageLink = subreddit.at(x).data.url
+		let postTitle = subreddit.at(x).data.title
+		let nsfw = subreddit.at(x).data.over_18
+		if (nsfw === true & message.channel.nsfw === false){
+			return message.reply(functions.simpleEmbed('Cannot post nsfw content in sfw channel!'))
+		}
+		const embed = new MessageEmbed()
```

Comment:
please take a look at how i've styled the embed for example for the urbandictionary command, as the current embed is pretty bland. i'd also recommend adding an `r/` in front of the title, for aesthetic reasons